README.md

```diff
@@ -149,15 +149,16 @@ RMT is replacing some functionality of [SMT](https://github.com/SUSE/smt). Follo
 |Red Hat support ([Expanded Support](https://www.suse.com/products/expandedsupport/))|:heavy_check_mark:|:x:<sup>[2](#res)</sup>|
 |Files deduplication|:heavy_check_mark:|:heavy_check_mark:|
 |Data transfer from SMT to RMT|-|:heavy_check_mark:|
-|Transfer registration data to SCC|:heavy_check_mark:|:x:|
+|Transfer registration data to SCC|:heavy_check_mark:|:x:<sup>[3](#regup)</sup>|
 |Reporting|:heavy_check_mark:|:x:|
 |Custom TLS certificates for web-server|:heavy_check_mark:|:heavy_check_mark:|
 |Webserver|Apache2|Nginx|
 |Database|MariaDB|MariaDB|
 |Platform|Perl|Ruby|
 
 <a name="staging">1</a>: Functionality is offered by [SUSE Manager](https://www.suse.com/documentation/suse-best-practices/susemanager/data/susemanager.html)  
-<a name="res">2</a>: RES support is planned for SLES15 SP1
+<a name="res">2</a>: RES support is planned for SLES15 SP1  
+<a name="regup">3</a>: Registartion data transfer to SCC planned for SLES15 SP2 or later
```

Comment:
'Registration'
s/planned/is planned/
s/or later/at the earliest/ ? Sounds more positive. Your call.